content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -86,14 +86,18 @@ In the above example, this version is `kubelet.config.k8s.io/v1beta1`.
 
 ## Drop-in directory for kubelet configuration files {#kubelet-conf-d}
```

Comment:
Beta graduation criteria ask to document the /configz endpoint as this is a recommended and only way for a cluster admin to review the resuting configuration file. Can you please send a PR here for this?

Link to graduation criteria: https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3983-drop-in-configuration/README.md#beta

Reply:
Sure, does it make sense to add that before the drop-in directory point with the title `Viewing the kubelet configuration`?

Reply:
this is the first mention of `/configz` in docs ever. I think it deserves it's own page. @sftim what would be the best place to document the `/configz` endopint?

Reply:
Hi. Best place would be within https://kubernetes.io/docs/reference/node/